### PR TITLE
Feature/additional arrays

### PR DESCRIPTION
**Pull request by Moin11786**
All of my assigned tasks are now coded apart from "going now".

### CODE REVIEW

File: talkitover_app.py [hunk 1]
```diff
@@ -655,37 +745,44 @@ def choose_bot_wordy_response(message, clientId):
...
-                        "i need to speak to a human", "i need to speak to a real person", "i need to speak to a real human" ]
+                        "i need to speak to a human", "i need to speak to a real person", "i need to speak to a real human", "is this all you do" ]
     thisBotIsBadArray_tight = ["you are bad", "you are awful", "you are terrible", "you are atrocious", "you are shit", "you are crap",
                         "this isnt helping", "this is not helping", "this aint helping",
                         "youre not helping", "your not helping", "you are not helping", "you arent helping", "you aint helping", "your no help", "youre no help", "your not a help", "youre not a help",
                         "this is useless", "this is worthless", "this is crap", "this is rubbish", "this is trash", "this is annoying", "this is pointless",
                         "you are useless", "you are worthless", "you are crap", "you are rubbish", "you are trash", "you are annoying", "you are pointless"
-                        "what a waste of time", "what a pointless waste of time", "what a useless waste of time"]
```
Comment (OliDeane) [inline talkitover_app.py:664]: This causes an error - I assume because the list is closed off twice.

File: talkitover_app.py [hunk 2]
```diff
@@ -497,12 +563,12 @@ def choose_bot_wordy_response(message, clientId):
     dontHaveAnyoneICanTalkToArray = ["don't have anyone i can talk to", "dont have anyone i can talk to","don't have anyone i can talk with", "dont have anyone i can talk with",
                                     "i have no one to talk to", "i have nobody to talk to", "i have no one to talk with", "i have nobody to talk with",
                                     "i've no one to talk to", "i've nobody to talk to", "i've no one to talk with", "i've nobody to talk with",
-                                    "ive no one to talk to", "ive nobody to talk to", "ive no one to talk with", "ive nobody to talk with", ]
```
Comment (OliDeane) [inline talkitover_app.py:500]: Causes an indentation error - list closed twice

File: talkitover_app.py [hunk 3]
```diff
@@ -497,12 +563,12 @@ def choose_bot_wordy_response(message, clientId):
...
     iHateHowILookArray = ["i hate how i look", "i hate my looks", "i hate my appearance", "i hate the way i look",\
                         "i just hate how i look", "i just hate my looks", "i just hate my appearance", "i just hate the way i look",\
                         "i really hate how i look", "i really hate my looks", "i really hate my appearance", "i really hate the way i look",\
                         "i really just hate how i look", "i really just hate my looks", "i really just hate my appearance", "i really just hate the way i look",
-                        "im ugly", "i am ugly", "i look horrible", "i look disgusting", "i look atrocious", "i feel insecure about my body", "i feel insecure about my appearance", "i hate my body", "i despise my body"]
+                        "im ugly", "i am ugly", "i look horrible", "i look disgusting", "i look atrocious", "i feel insecure about my body", "i feel insecure about my appearance", "i hate my body", "i despise my body", "i am so humiliated that i look like this", "i feel so humiliated that i look like this"]
```
Comment (OliDeane) [inline talkitover_app.py:571]: Indentation error

File: talkitover_app.py [hunk 4]
```diff
@@ -638,9 +721,16 @@ def choose_bot_wordy_response(message, clientId):
                             "have you any advice", "have you some advice", "have you advice"
                             "give me advice", "i need advice",
                             "dont you give advice", "you dont give advice",
-                            "will you give me advice", "will you give advice"]
```
Comment (OliDeane) [inline talkitover_app.py:641]: indentation error

File: talkitover_app.py [hunk 5]
```diff
@@ -826,7 +977,7 @@ def cleanText(message):
         """
 
         extraneousWordsArray = ["still", "just", "so", "very", "totally", "utterly", "really", "completely", \
-        "literally", "actually", "even", "some", "always", "fucking", "fuckin"]
+        "literally", "actually", "even", "some", "always", "fucking", "fuckin", "technically"]
```
Comment (OliDeane) [inline talkitover_app.py:980]: Error - I guess delete the duplicated line?

**Review by OliDeane — COMMENTED** [review]
I got  couple of errors when I tried to run this. I think I have commented on most of them. They're only small (and I'm not sure if they are unique to this script) - I just thought it would be best to resolve them before merging with the develop branch.

**Comment by Moin11786** [comment]
Closing to fix issues - I will create another one.

**Pull request closed without merging**